### DISCUSSION TITLE
Xdebug PHP CLI script

### DISCUSSION THREAD

**Discussion by chombe** [markshust/docker-magento#482] [Q&A]
Hello Mark.

Xdebug works great for web requests.
Do you maybe know what configuration is needed for getting Xdebug to work with a CLI PHP scrip ?
For example when running php bin/magento some:script

Thanks in advance

**Comment by rangerz** [comment]
It may be helpful.
https://www.jetbrains.com/help/phpstorm/debugging-a-php-cli-script.html

**Reply by radub** [reply]
@rangerz could you please elaborate a bit? I'm pretty familiar with xdebug but I can't seem to make this setup work in the CLI mode.
